src/containers/AddUsersModal/index.jsx

```diff
@@ -0,0 +1,34 @@
+import { connect } from 'react-redux';
+
+import AddUsersModel from '../../components/AddUsersModal';
+
+import subscribeUsers from '../../data/actions/userSubscription';
+import {
+  clearSubscriptionDetails,
+  fetchSubscriptionDetails,
+} from '../../data/actions/subscriptionDetails';
+
+const mapStateToProps = state => ({
+  enterpriseId: state.portalConfiguration.enterpriseId,
+  initialValues: state.emailTemplate.subscribe,
+  availableSubscriptionCount: state.subscriptionDetails.licenses.available,
```

Comment:
`available` is the total number of licenses purchases for a SubscriptionPlan. `allotted` is how many have been assigned/activated. Thus, the number of available licenses that an enterprise admin could assign to their employees is `available - allotted`.

For example, my SubscriptionPlan gives me 500 licenses. I've assigned 100 of those licenses. I have 400 remaining ones that I can assign.

Also: `availableSubscriptionCount` --> `availableLicensesCount `